components/member-profile/contribution/index.js

```diff
@@ -68,9 +113,16 @@ const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
         </div>
       </div>
       <div className={classNames.featureLink}>
-        <a href={featureUrl || prList[0]['url']} target="_blank" rel="noreferrer">
-          Check out this feature in action
-        </a>
+        {url &&
+          (urlObj.host === 'members.realdevsquad.com' ? (
```

Comment:
Nit: Abstract this into a `renderFeatureURL()` to decrease logic in return code path